appserver/security/core-ee/src/main/java/com/sun/enterprise/security/acl/RoleMapperFactory.java

```diff
@@ -46,73 +46,86 @@
 import java.util.HashMap;
 import java.util.Hashtable;
 
-
 import org.jvnet.hk2.annotations.Service;
 import javax.inject.Singleton;
 
 /**
  *
- * @author  Jerome Dochez
+ * @author Jerome Dochez
```

Comment:
Copyright needs adding

Reply:
I was a bit hesitant to add them, since some files are just formatting, others have more changes. I'll take a look though.